File: JAVA/Ejercicio1.java
import java.util.Scanner;
public class Ejercicio1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Introduce la distancia en kilometros: ");
        int d=sc.nextInt();
        System.out.print("Introduce la velocidad en kilometros por hora: ");
        int v=sc.nextInt();
        int t=d/v;
        System.out.println("El tiempo total es de: "+t+" horas");
    }
}

File: PYTHON/Ejercicio1.py
'''
Escribe un programa que calcule el tiempo que tardará un vehículo en recorrer una distancia a una velocidad constante. 
El programa debe pedir al usuario que ingrese la distancia en kilómetros y la velocidad en kilómetros por hora.
Mostrar el tiempo que tardará en recorrer la distancia en horas.
'''
d=int(input("Introduce la distancia en kilometros: "))
v=int(input("Introduce la velocidad en kilometros por hora: "))
t=d/v
print("El tiempo total es de:",t,"horas")